Tests.Unit.Core.ApplicationServices/ApplicationServices/Deltas/TrackingServiceTests.cs

```diff
@@ -0,0 +1,205 @@
+            all.Take(1).ToList().ForEach(x => x.OptionalOrganizationReference.Id = orgIds.First());
+            all.Skip(1).Take(1).ToList().ForEach(x => x.OptionalOrganizationReference.Id = orgIds.Last());
+            all.Skip(2).Take(1).ToList().ForEach(x => x.OptionalAccessModifier = AccessModifier.Public);
+
+            _organizationUserContextMock.Setup(x => x.OrganizationIds).Returns(orgIds);
+            _repositoryMock.Setup(x => x.AsQueryable()).Returns(all);
+            ExpectCrossLevelOrganizationReadAccess(CrossOrganizationDataReadAccessLevel.None);
+
+            //Act
+            var result = _sut.QueryLifeCycleEvents();
+
+            //Assert - first 2 by org id relation, - nothing else - not even the shared item at index 2
+            Assert.Equal(all.Take(2), result);
+        }
+
+        [Fact]
+        public void Can_Query_With_No_Refinements_As_RightsHolderAccess()
+        {
+            //Arrange
+            var uniqueIds = CreateUniqueIds(13).ToList();
+            var orgIds = uniqueIds.Take(2).ToList();
+            var rightsHolderOrg = uniqueIds.Skip(2).First();
+            var idsToSetOnFullSet = new Stack<int>(uniqueIds.Skip(3));
+            var all = Many<LifeCycleTrackingEvent>(10).AsQueryable();
+
+            all.ToList().ForEach(x =>
+            {
+                var id = idsToSetOnFullSet.Pop();
+                x.OptionalOrganizationReference.Id = id;
+                x.OptionalRightsHolderOrganization.Id = id;
+            });
+            all.Take(1).ToList().ForEach(x => x.OptionalOrganizationReference.Id = orgIds.First());
+            all.Skip(1).Take(1).ToList().ForEach(x => x.OptionalOrganizationReference.Id = orgIds.Last());
+            all.Skip(2).Take(1).ToList().ForEach(x => x.OptionalAccessModifier = AccessModifier.Public);
+            all.Skip(3).Take(1).ToList().ForEach(x => x.OptionalRightsHolderOrganization.Id = rightsHolderOrg);
+
+            _organizationUserContextMock.Setup(x => x.OrganizationIds).Returns(orgIds);
+            _organizationUserContextMock.Setup(x => x.GetOrganizationIdsWhereHasRole(OrganizationRole.RightsHolderAccess)).Returns(new[] { rightsHolderOrg });
+            _repositoryMock.Setup(x => x.AsQueryable()).Returns(all);
+            ExpectCrossLevelOrganizationReadAccess(CrossOrganizationDataReadAccessLevel.RightsHolder);
+
+            //Act
+            var result = _sut.QueryLifeCycleEvents();
+
+            //Assert - first 2 by org id relation, then at index 3 the rights holder org
+            Assert.Equal(all.Take(2).Concat(all.Skip(3).Take(1)), result);
+        }
+
+
+        public static IEnumerable<object[]> GetTrackedEntityTypes() => EnumRange.All<TrackedEntityType>().Select(x => new object[] { x });
+
+        [Theory, MemberData(nameof(GetTrackedEntityTypes))]
+        public void Can_Query_With_EntityTypeFilter(TrackedEntityType filterBy)
```

Comment:
Hvad med en test på TrackedLifeCycleEventType? 

Mener kun det er Delete vi har lige nu, men den er vel stadig relevant?

Reply:
ja jeg kan bare ikke rigtig teste nogen effekt for det vil bare være "det hele". Jeg kan selvfølgelig godt lave en test, som kan være klar til udvidelse